.dockerignore

```diff
@@ -0,0 +1,4 @@
+.env
+.git
+.vscode
+.venv
```

Comment:
This covers the key things, but worth considering what else might be worth ignoring. For example:
* `Dockerfile` - currently any time your Dockerfile changes, you end up invalidating the build cache; so this can be a small performance improvement (particularly helpful for just speeding up development)
* Any cache folders that you see, e.g. `**/,pytest_cache` and `**/__pyache__`
* `README.md`?